doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -659,12 +659,14 @@ file is a particular case of a module called a *library file*.
 
 .. cmd:: Declare ML Module {+ @string }
 
-   Loads an OCaml plugin and its dependencies dynamically.  The :n:`@string`
-   argument must be a valid `findlib <http://projects.camlcity.org/projects/findlib.html>`_
-   plugin name, for example ``coq-core.plugins.ltac``. As of Coq 8.16,
-   the command also supports a legacy
-   syntax compatible with the plugin loading system used in Coq
-   8.0-8.15, see below.
+   Loads OCaml plugins and their dependencies dynamically.  The :n:`@string`
+   arguments must be valid `findlib <http://projects.camlcity.org/projects/findlib.html>`_
+   plugin names, for example ``coq-core.plugins.ltac``.
+   They must also contain a `.` to disambiguate against the removed non-findlib loading method.
```

Comment:
What happens if they don't contain a `.`?  Do you get an error message or the user has to remember?

Reply:
error

Reply:
Can't this actually be relaxed with this MR, given that there does not seem to be anything to disambiguate anymore?

Reply:
I propose that we defer this change to another PR. It's not currently used, meanwhile the legacy loading mechanism has passed its shelf life by a large amount. The sooner we get rid of it the better, and the attempted branch date for 9.0 is coming close.

Reply:
Note that this is a problem for some users, as there are findlib libraries that don't contain a `.`.

So IMHO, this is a RC bug.